### PR TITLE
restructure

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 14 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-988163148). valid for 14 days

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 15 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-988163148). valid for 14 days

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 16 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-988163148). valid for 14 days

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 17 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-ce71777c7). valid for 14 days

File: docs/conf.py [hunk 1]
```diff
@@ -1,20 +1,18 @@
 # -*- coding: utf-8 -*-
 """
-Rackspace Colocation Guide build configuration file.
+Digital Experience Customer Handbook build configuration file.
```
Comment (stephamon) [inline docs/conf.py:3]: Probably want to put this back to the colo guide :)

**Comment by rackernexus** [comment]
Build 18 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-c5b95c295). valid for 14 days

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 19 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-c5b95c295/docs/docs-colocation-guide/). valid for 14 days

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 20 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-b4333032f/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 21 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-b4333032f/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 22 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-fb8cd4c74/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 23 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-5e3c31e02/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 24 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-bffe3f231/docs/docs-colocation-guide/). valid for 14 days

**Comment by WilliamLoy** [comment]
build me

**Comment by rackernexus** [comment]
Build 26 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-a7ccb8655/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 27 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-bcae633c3/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 28 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-f443e5ff0/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 29 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-b250d4764/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 30 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-df1e4bd3b/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 31 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-48abd3e6a/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 32 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-62cb19f62/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 33 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-d9a8db5e9/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 34 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-a8b5f43a9/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 35 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-26091ff92/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 36 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-d97fdb604/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 37 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-f4a7f578c/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 38 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-4aef22137/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 39 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-c41c5b5a2/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 40 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-5102f1ff6/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 41 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-4c5515c7b/docs/docs-colocation-guide/). valid for 14 days

**Comment by rackernexus** [comment]
Build 42 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-249f2b8f1/docs/docs-colocation-guide/). valid for 14 days

**Pull request merged**